dotrix_egui/src/lib.rs

```diff
@@ -19,7 +19,7 @@ use dotrix_core::{Application, Assets, Id, Input, Pipeline, Window};
 
 use dotrix_overlay::{Overlay, Ui, Widget};
 
-pub use egui::*;
+pub use egui::{self, *};
```

Comment:
lets re-export `self` as `native` so we can use it as `use dotrix::egui::native::Ui`?

Reply:
Done